src/main/java/com/sprint/mission/discodeit/repository/file/FileChannelRepository.java

```diff
@@ -0,0 +1,69 @@
+package com.sprint.mission.discodeit.repository.file;
+
+import com.sprint.mission.discodeit.entity.Channel;
+import com.sprint.mission.discodeit.repository.ChannelRepository;
+
+import java.io.*;
+import java.util.*;
```

Comment:
와일드 카드 import는 지양하는게 좋습니다.  

[구글 Java 스타일 가이드](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)에도 와일드카드 import에 대한 내용이 적혀 있습니다.  

> 3.3.1 No wildcard imports  
Wildcard imports, static or otherwise, **are not used.**  


클래스를 봤을 때 의존성이 어떻게 되어있는지 import문을 통해 육안으로 확인할 수 있는데요,  
와일드카드를 사용하게 되면 한눈에 의존성을 확인할 수 없게 됩니다.

혹시라도 인텔리제이에 자동으로 설정되어 있다면 자동 import 설정을 바꾸는게 좋을 것 같아요! [참고 링크](https://hianna.tistory.com/835)